.github/workflows/deploy-master.yml

```diff
@@ -4,10 +4,15 @@ on:
         branches: [master]
 jobs:
     deploy:
-        runs-on: self-hosted-novum
-        container: docker.tuenti.io/service-inf/web-builder:pptr10.4-1.0.2
+        runs-on: ubuntu-latest
+        container: telefonicacdo/web-builder:chromium93.0.4577-1.2.3
         steps:
             - uses: actions/checkout@v2
+
+            - name: install-vercel
+              # workaround for https://github.com/vercel/vercel/issues/6811
+              run: npm i -g vercel@canary
```

Comment:
is this a temporal fix? can you create a ticket to revert it?

Reply:
https://jira.tid.es/browse/WEB-417